content/applications/marketing/marketing_automation.rst

```diff
@@ -1,121 +1,186 @@
+Campaigns
+=========
 
-In the :guilabel:`Target` field, use the drop-down menu to choose which model the target audience
-filters should be based on (e.g. :guilabel:`Contact`, :guilabel:`Lead/Opportunity`,
-:guilabel:`Sales Order`, etc.).
+A *campaign* consists of a target audience and a workflow containing activities that are
+automatically executed to a specific group of records based on predefined filters, triggers and
+durations of activities.
 
-Select :guilabel:`Search More...` from the drop-down menu to reveal a :guilabel:`Search: Target`
-pop-up window containing all of the available targeting options.
+A new campaign can be created from scratch or from a template. To create a campaign, navigate to the
+:menuselection:`Marketing Automation` application from the main Odoo dashboard to open the
+:guilabel:`Campaigns` dashboard. From here, click the :guilabel:`New` button to reveal a new
+campaign form.
 
-Once a :guilabel:`Target` is selected, there's a :guilabel:`Unicity based on` field. This field is
-used to avoid duplicates based on the model chosen in the :guilabel:`Target` field.
+.. _marketing-automation/campaign-templates:
 
-.. example::
-   If :guilabel:`Customers` is chosen as the :guilabel:`Target`, select :guilabel:`Email` in the
-   :guilabel:`Unicity based on` field so Odoo only processes one record for each customer email
-   address.
+Campaign templates
+------------------
 
-Select :guilabel:`Search More...` from the :guilabel:`Unicity based on` drop-down menu to reveal all
-available options in a pop-up window.
+Odoo provides six campaign templates to help users get started. The campaign template cards only
+display while there are no existing campaigns in the database. Once a campaign has been created, the
+template cards on the *Campaigns* dashboard are replaced with a Kanban view of the existing
+campaigns.
 
-Last on the campaign form is the :guilabel:`Filter` field. This is where more specific targeting
-options can be layered into the campaign to further narrow the number and type of recipients that
-receive the marketing automation material.
+To get started with a template, navigate to the :menuselection:`Marketing Automation` application
+from the main Odoo dashboard to open the :guilabel:`Campaigns` dashboard, which displays six
+campaign template cards:
 
-If left alone, the :guilabel:`Filter` field reads: :guilabel:`Match all records`. That means Odoo
-uses the :guilabel:`Target` and :guilabel:`Unicity based on` fields to determine who the recipients
-will be. The number of recipients is represented beneath as :guilabel:`record(s)`.
+- | :guilabel:`Tag Hot Contacts`
+  | :guilabel:`Send a welcome email to contacts and tag them if they click it.`
+- | :guilabel:`Welcome Flow`
+  | :guilabel:`Send a welcome email to new subscribers, remove the address that bounced.`
+- | :guilabel:`Double Opt-in`
+  | :guilabel:`Send an email to new recipients to confirm their consent.`
+- | :guilabel:`Commercial prospection`
```

Comment:
I am going to argue that when using a :guilabel: the text should match the UI exactly, with the exception of dynamic text (like # numbers inside of buttons for example). I can create a feedback ticket for this fix though because I agree it seems like a mistake/inconsistency!